pkg/controller/bootstrap/jws_test.go

```diff
@@ -26,42 +30,29 @@ const (
 
 func TestComputeDetachedSig(t *testing.T) {
 	sig, err := computeDetachedSig(content, id, secret)
-	if err != nil {
-		t.Errorf("Error when computing signature: %v", err)
-	}
-	if sig != "eyJhbGciOiJIUzI1NiIsImtpZCI6Impvc2h1YSJ9..VShe2taLd-YTrmWuRkcL_8QTNDHYxQIEBsAYYiIj1_8" {
-		t.Errorf("Wrong signature. Got: %v", sig)
-	}
+	assert.NoError(t, err, "Error when computing signature: %v", err)
+	assert.Equal(t, "eyJhbGciOiJIUzI1NiIsImtpZCI6Impvc2h1YSJ9..VShe2taLd-YTrmWuRkcL_8QTNDHYxQIEBsAYYiIj1_8", sig, "Wrong signature. Got: %v", sig)
```

Comment:
It's too long, please make it shorter. Same with or other places.

Reply:
Done